transcom/mymove
# B 21542 int - DoD ID Does Not Populate in Services Counselor Queue

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21542)

## Summary
When office users login to MM, their queue is not showing the DOD ID of the customer.


### How to test
1. Login as a customer and create a move.
2. Login as a  Services Counselor verify the DoD ID value is displayed in the counseling Queue
3. submit the move
4. Login as TOO,  verify the DoD ID value is displayed In the task order queue
5. approve the move and create a payment request
6. Login as a or TIO verify the DoD ID value is displayed In the payment request queues


## Screenshots
